clang/lib/AST/ASTContext.cpp

```diff
@@ -1390,7 +1390,8 @@ void ASTContext::InitBuiltinTypes(const TargetInfo &Target,
 #include "clang/Basic/HLSLIntangibleTypes.def"
   }
 
-  if (Target.hasAArch64SVETypes()) {
+  if (Target.hasAArch64SVETypes() ||
```

Comment:
Is this expected change for intangible AST type?

Reply:
I don't see this change in this PR diff. This change was done in llvm/llvm-project#99446 and came in via merge from `main`.